python/llm/dev/benchmark/harness/make_table_results.py

```diff
@@ -67,10 +69,39 @@ def make_table(result_dict):
 
     return md_writer.dumps()
 
+def make_csv(result_dict, output_path=None):
+    current_date = datetime.datetime.now().strftime("%Y-%m-%d")
+    file_name = f'results_{current_date}.csv'
+    full_path = os.path.join(output_path, file_name) if output_path else file_name
+    print('Writing to', full_path)
```

Comment:
I don't think `make_csv` is related to this file? So maybe you need a new file to run this. Otherwise you should carefully modify this file and its name.

Reply:
modified the file and its name.